src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -268,6 +272,16 @@ static MvcCoreLoggerExtensions()
                 LogLevel.Debug,
                 3,
                 "The request body size limit has been disabled.");
+
+            _cannotApplyRequestFormLimits = LoggerMessage.Define(
+                LogLevel.Warning,
+                1,
+                "Could not apply limits to the request form as it might have been read already.");
```

Comment:
This includes `BufferBody` which is not exactly a limit. Should this say 

> Unable to apply requested form options since the request form has already been read."